File: dastanlib/src/main/java/com/dastanapps/dastanlib/receiver/UpdateApp.java
package com.dastanapps.dastanlib.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;


public class UpdateApp extends BroadcastReceiver {
    public UpdateApp() {
    }

    @Override
    public void onReceive(Context context, Intent intent) {
        SendBroadcast.AppUpdate(intent);
    }
}

File: common/src/main/java/com/dastanapps/dastanlib/receiver/UpdateApp.kt
package com.dastanapps.dastanlib.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent


class UpdateApp : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        SendBroadcast.AppUpdate(intent)
    }
}
